jetty-core/jetty-websocket/jetty-websocket-core-server/src/main/java/org/eclipse/jetty/websocket/core/server/internal/ServerUpgradeRequestImpl.java

```diff
@@ -14,27 +14,19 @@
 package org.eclipse.jetty.websocket.core.server.internal;
 
 import java.util.List;
-import java.util.Set;
 
 import org.eclipse.jetty.http.BadMessageException;
 import org.eclipse.jetty.http.HttpHeader;
 import org.eclipse.jetty.server.Request;
-import org.eclipse.jetty.util.Attributes;
 import org.eclipse.jetty.websocket.core.ExtensionConfig;
 import org.eclipse.jetty.websocket.core.WebSocketComponents;
 import org.eclipse.jetty.websocket.core.WebSocketConstants;
 import org.eclipse.jetty.websocket.core.server.ServerUpgradeRequest;
 
-/**
- * Upgrade request used for websocket negotiation.
- * Provides getters for things like the requested extensions and subprotocols so that the headers don't have to be parsed manually.
- */
 public class ServerUpgradeRequestImpl extends Request.Wrapper implements ServerUpgradeRequest
```

Comment:
Javadoc

Reply:
@lachlan-roberts  The methods themselves don't need javadoc.  Just the class.  A single phrase saying how/when/where/who the class is used will suffice.